src/Logging/V2/ConfigServiceV2Client.php

```diff
@@ -268,14 +266,14 @@ public function __construct($options = [])
         $this->scopes = $options['scopes'];
 
         $createStubOptions = [];
-        if (!empty($options['sslCreds'])) {
+        if (array_key_exists('sslCreds', $options)) {
             $createStubOptions['sslCreds'] = $options['sslCreds'];
         }
         $grpcCredentialsHelperOptions = array_diff_key($options, $defaultOptions);
         $this->grpcCredentialsHelper = new GrpcCredentialsHelper($this->scopes, $grpcCredentialsHelperOptions);
 
         $createConfigServiceV2StubFunction = function ($hostname, $opts) {
-            return new ConfigServiceV2Client($hostname, $opts);
+            return new \google\logging\v2\ConfigServiceV2Client($hostname, $opts);
```

Comment:
You could also alias this in the namespace declarations above to avoid collision:

`use google\logging\v2\ConfigServiceV2Client as ProtoClient;`